examples/text_classification-ort.ipynb

```diff
@@ -0,0 +1,1289 @@
+    "\n",
+    "Here, instead of `Trainer`, you will use the `ORTTrainer` class in [🏎️ Optimum ](https://github.com/huggingface/optimum) library and take [ONNX Runtime](https://microsoft.github.io/onnxruntime/) as backend to accelerate the training. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9408e05-175b-4573-8314-9ae4ba7d3453",
+   "metadata": {},
+   "source": [
+    "__Dependencies__\n",
+    "\n",
+    "To use ONNX Runtime for training, you need a machine with at least one NVIDIA or AMD GPU.\n",
+    "\n",
+    "CUDA and cuDNN need to be properly installed, before the installation of ONNX Runtime training module. Please follow the instruction in [Optimum's documentation](https://huggingface.co/docs/optimum/onnxruntime/trainer) to set up your environment.\n",
+    "\n",
+    "Check your GPU:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "74c87883-595d-47cb-acf9-091340705b33",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fri Sep 16 09:45:13 2022       \n",
+      "+-----------------------------------------------------------------------------+\n",
+      "| NVIDIA-SMI 440.33.01    Driver Version: 440.33.01    CUDA Version: 11.3     |\n",
+      "|-------------------------------+----------------------+----------------------+\n",
+      "| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |\n",
+      "| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |\n",
+      "|===============================+======================+======================|\n",
+      "|   0  Tesla T4            On   | 00000000:00:1E.0 Off |                    0 |\n",
+      "| N/A   27C    P8     8W /  70W |      0MiB / 15109MiB |      0%      Default |\n",
+      "+-------------------------------+----------------------+----------------------+\n",
+      "                                                                               \n",
+      "+-----------------------------------------------------------------------------+\n",
+      "| Processes:                                                       GPU Memory |\n",
+      "|  GPU       PID   Type   Process name                             Usage      |\n",
+      "|=============================================================================|\n",
+      "|  No running processes found                                                 |\n",
+      "+-----------------------------------------------------------------------------+\n"
+     ]
+    }
+   ],
+   "source": [
+    "!nvidia-smi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e379d8ab-9495-4c2a-ace9-f1e605ea9adf",
+   "metadata": {},
+   "source": [
+    "If you're opening this Notebook on colab, you will probably need to install 🤗 Optimum, 🤗 Transformers, 🤗 Datasets and 🤗 evaluate. Uncomment the following cell and run it."
```

Comment:
Probably we should hint to install `optimum[onnnxruntime-gpu]`? And also `torch_ort`?

Reply:
I actually won't recommend users to set up the prerequisite(including onnx runtime dependencies and torch_ort) manually in the notebook, I prefer to let them set it up first either locally or with the docker files that we offer, and then install optimum and transformers, etc. in the notebook. I will make it more clear in the document that they need to set up the env first.